proof_of_concept/local_workflow_runner.py

```diff
@@ -174,6 +179,13 @@ def _get_step_inputs(
 
         return step_input_data
 
+    def _retrieve_compute_asset(self, compute_asset_name: str) -> ComputeAsset:
+        # TODO: the store id is hardcoded so scenario pii test works,
```

Comment:
@LourensVeen would be nice to discuss this

Reply:
We decided to make a hardcoded solution for now (that works for all tests) i.e. have a ddm which acts as a central repository of software.

Reply:
And then solve this in https://dev.azure.com/nlesc-team-flow/Team%20FLOW/_workitems/edit/640

Reply:
Correction: instead of hardcoded store_id we can use `get_asset_location`.